### ISSUE TITLE
Is it possible to enable zoom feature for pieChart

### ISSUE THREAD

**Issue by srigurajala** [krispo/angular-nvd3#300]
Currently 6/7 graphs have zoom support in latest version [1.0.5], but i want to have zoom options for all graphs including pieChart. Please suggest how to enable the zoom feature to pieChart as well

**Comment by GhaneshyamSolanki** [comment]
Yes it will be good for piechart, barchart and multi bar charts....+1
